iteratees/src/main/scala/play/api/libs/iteratee/Iteratee.scala

```diff
@@ -695,36 +686,26 @@ trait Iteratee[E, +A] {
  * state is immediately available.
  */
 private sealed trait StepIteratee[E, A] extends Iteratee[E, A] with Step[E, A] {
-
   final override def it: Iteratee[E, A] = this
   final def immediateUnflatten: Step[E, A] = this
 
-  final override def unflatten: Future[Step[E, A]] = Future.successful(immediateUnflatten)
+  final override def unflatten: Future[Step[E, A]] =
+    Future.successful(immediateUnflatten)
 
   final override def fold[B](folder: Step[E, A] => Future[B])(implicit ec: ExecutionContext): Future[B] = {
     executeFuture {
       folder(immediateUnflatten)
     }(ec /* executeFuture handles preparation */ )
   }
 
-  final override def pureFold[B](folder: Step[E, A] => B)(implicit ec: ExecutionContext): Future[B] = {
-    Future {
-      folder(immediateUnflatten)
-    }(ec /* Future.apply handles preparation */ )
-  }
+  final override def pureFold[B](folder: Step[E, A] => B)(implicit ec: ExecutionContext): Future[B] = Future(folder(immediateUnflatten))(
```

Comment:
If you want to avoid braces for the method body, this could be better written as:

```scala
final override def pureFold[B](folder: Step[E, A] => B)(implicit ec: ExecutionContext): Future[B] =
  Future { folder(immediateUnflatten) }
```

Same thing for the rest of the methods you changed here. It's more readable to have the body on the next line.